clouddrift/wavelet.py

```diff
@@ -15,22 +15,52 @@
 not the MATLAB implementation is licensed under CloudDrift's MIT license.
 """
 
+import typing as t
+
 import numpy as np
 from scipy.special import gamma as _gamma
 from scipy.special import gammaln as _lgamma
 
 
+@t.overload
+def morse_wavelet_transform(
+    x: np.ndarray,
+    gamma: float,
+    beta: float,
+    radian_frequency: np.ndarray,
+    complex: t.Literal[True],
+    order: int = 1,
+    normalization: str = "bandpass",
+    boundary: str = "mirror",
+    time_axis: int = -1,
+) -> tuple[np.ndarray, np.ndarray]: ...
+
+
+@t.overload
+def morse_wavelet_transform(
+    x: np.ndarray,
+    gamma: float,
+    beta: float,
+    radian_frequency: np.ndarray,
+    complex: t.Literal[False],
+    order: int = 1,
+    normalization: str = "bandpass",
+    boundary: str = "mirror",
+    time_axis: int = -1,
+) -> np.ndarray: ...
+
+
 def morse_wavelet_transform(
     x: np.ndarray,
     gamma: float,
     beta: float,
     radian_frequency: np.ndarray,
-    complex: bool = False,
+    complex: bool,
```

Comment:
a boolean is False by default? Why remove the default?

Reply:
I can revert this change.